web/ui_utils.js

```diff
@@ -105,6 +105,41 @@ function getOutputScale(ctx) {
 }
 
 /**
+ * @param {number} The width of a page, in points.
+ * @param {number} The height of a page, in points.
+ * @return {Object} An object with the width and height properties which is are
+ *   numbers with a CSS unit.
+ */
+function calculatePageDimensions(width, height) {
+  // Browsers (at least Firefox and Chrome) use pixels to measure the (print)
+  // canvas size. PDFs specify the dimensions of a canvas in points.
+  // 96px = 72pt (http://www.w3.org/TR/css3-values/#absolute-lengths)
+
+  // Browsers are free to round fractions in any way they want. If the nearest
+  // integer algorithm is used, then an A4 would be 1123x793 px, which results
+  // in one logical page that is split over two printed pages.
+  // To make sure that the page fits on printed paper, points are converted to
+  // pixels, the width is rounded UP, while the height is rounded DOWN.
+
+  var cssPixelsPerPoint = 96 / 72;
```

Comment:
Since this is a constant, please change the name to `CSS_PIXELS_PER_POINT`.
Also, since the value is identical to `CSS_UNITS`, could we just use that one instead?